examples/mobile/ScheduleApp/js/day-indicator.js

```diff
@@ -0,0 +1,67 @@
+class DayControl extends HTMLElement {
+
+	connectedCallback() {
+		this.classList.add("day-control");
+	}
+}
+
+customElements.define("day-control", DayControl);
+
+class DayIndicator extends HTMLElement {
+
+	constructor() {
+		super();
+		this.daysLabels = ["S", "M", "T", "W", "T", "F", "S"];
+	}
+
+	connectedCallback() {
+		this.classList.add("day-indicator");
+		this.addDays();
+		this.updateActiveDays();
+	}
+
+	attributeChangedCallback(attrName) {
+		if (attrName == "active") {
+			this.updateActiveDays();
+		}
+	}
+
+	get active() {
+		return this.getAttribute("active");
+	}
+
+	set active(value) {
+		this.setAttribute("active", value);
+		this.updateActiveDays();
+	}
+
+	addDays() {
```

Comment:
I think this is good practice to add "_" prefix before protected methods, eg.
https://developers.google.com/web/fundamentals/web-components/examples/howto-checkbox